language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/docs/DocumentationGenerator.java

```diff
@@ -285,9 +295,25 @@ private static DocAttachmentInfo getFunctionNodeDocumentation(FunctionSignatureN
             paramName.ifPresent(token -> parameters.put(token.text(), "Parameter Description"));
         });
         String returnDesc = signatureNode.returnTypeDesc().isPresent() ? "Return Value Description" : null;
-        if (hasDeprecated) {
-//            attributes.add(getDeprecatedDescription(offset));
+//        String deprecated = null;
+//        if (hasDeprecated) {
+//            deprecated = "This function is deprecated.";
+//        }
```

Comment:
Is is intentional?

Reply:
Yes kept it since we do need to support deprecated doc blocks and it requires changes from the Compiler as well. I'll add a TODO comment here.